src/zc/lockfile/README.txt

```diff
@@ -68,3 +68,23 @@ If you now inspected the lock file, you would see e.g.:
     $ cat lock
      123;myhostname
 
+Context Managers
+================
+
+This module also implements two Context Managers, in addition to the primitives
+discussed above. The first just gets and releases a lock file, with a timeout.
+
+    >>> rough_seconds = 0.7
+    >>> with zc.lockfile.LockedContext('description', timeout=rough_seconds):
+    ...    pass
+
+The lock will always be released when you exit this scope. If timeout is not
+given, it is assumed you wish to wait forever. It should be noted that this
```

Comment:
It might be nice to mention what kind of exception is raised if the timeout occurs.